training/src/main/scala/com/airbnb/aerosolve/training/FullRankLinearTrainer.scala

```diff
@@ -174,14 +166,15 @@ object FullRankLinearTrainer {
       gradient.iterator
     })
     .reduceByKey((a, b) => sumGradients(a,b))
```

Comment:
should we call GradientUtils.sumGradients(a,b) to be consistent and more clear about where the function come from ?